frontend/src/component/Task/ TaskView.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+interface Props {
+  array: string[];
+}
+
+
+const TaskModal: React.VFC<Props> = (props) => {
+
+  return(
+    <div className="task--view">
+        {
+        props.array.map((value) => 
+        <p>{value}</p>)
+        }
+    </div>
+  )
+}
```

Comment:
```suggestion
const TaskModal: React.VFC<Props> = ({array}) => {

  return(
    <div className="task--view">
        {
        array.map((value) => 
        <p>{value}</p>)
        }
    </div>
  )
}
```

関数の引数にpropsではなく、arrayを直接指定の方がいいと思う。
あと変数名がarrayだとプログラム言語の予約語と被ってる可能性があるのと一目で何の変数か理解ができないので、taskList またが taskArray といったふうに変数を指定したほうがソースコードの可読性が上がる。